themes/classic/templates/catalog/category.tpl

```diff
@@ -16,7 +16,7 @@
     {/block}
 
     {block name='category_products'}
-      {include file='catalog/_partials/products.tpl' products=$products}
+      {include file='catalog/_partials/products.tpl' products=$products label=$category.name}
```

Comment:
because you look at `category.tpl`, but if you look under, in `new-products.tpl` or `prices-drop/.tpl` you'll see :wink: